lib/board.ex

```diff
@@ -28,10 +22,10 @@ defmodule Board do
     IO.puts "\n"
   end
 
-  def update(board, list, list2, token, token2) do
+  def render(board, list, list2, token, token2) do
```

Comment:
These argument names are not great. I get `board`, and `token`/`token2` is ok, but what is `list`/`list2`?